openmp/runtime/test/tasking/omp_record_replay_deps_multi_succ.cpp

```diff
@@ -0,0 +1,60 @@
+// REQUIRES: ompx_taskgraph
+// RUN: %libomp-cxx-compile-and-run
+#include <omp.h>
+#include <cassert>
+#include <vector>
+
+#define TASKS_SIZE 12
+
+typedef struct ident {
```

Comment:
It doesn't need to have a definition. A forward declaration would be sufficient.